src/plugins/es_ui_shared/docs/docs/form_lib/examples/validation.md

```diff
@@ -0,0 +1,369 @@
+---
+id: validation
+title: Validation
+sidebar_label: Validation
+---
+
+## Basic
+
+```js
+import React from 'react';
+import {
+  useForm,
+  Form,
+  UseField,
+  FieldConfig,
+} from '<path-to-form-lib>';
+
+interface MyForm {
+  name: string;
+}
+
+const nameConfig: FieldConfig<MyForm, string> = {
+  validations: [
+    {
+      validator: ({ value }) => {
+        if (value.trim() === '') {
+          return {
+            message: 'The name cannot be empty.',
+          };
+        }
+      },
+    },
+    // ...
+    // You can add as many validations as you need.
+    // It is better to kepp validators single purposed.
```

Comment:
I mean validating a single type of error. To avoid creating a gigantic validator that can't be reused anywhere.

```js
if (value.length === 0) {
  return { message: 'The field cannot be empty' }
}

if (value.contains('/') {
  return { message: 'The char "/" is not allowed' }
}

...
```

Reply:
I added a comment to explain the good practice.